tools/rpkg/tests/testthat/test_extensions.R

```diff
@@ -0,0 +1,37 @@
+library("DBI")
+library("testthat")
+
+test_that("extensions can be loaded", {
+  con <- dbConnect(duckdb::duckdb())
+  on.exit(dbDisconnect(con, shutdown = TRUE))
+  matches = list.files(pattern='*.duckdb_extension', recursive=FALSE, full.names=TRUE)
```

Comment:
Maybe `<-` instead of `=` for consistency?

This line seems to be searching files from the folder where testthat tests will be runned? Could searching be expanded so that it would search also from the folder of installed DuckDB R package? Maybe additional line with something like:  `matches <- c(matches, list.files(path=system.file(package = 'duckdb'), pattern='*.duckdb_extension', recursive=TRUE, full.names=TRUE))` The point would be that maybe it could be possible to include compiled extensions to the R package inst/ directory at least for the Mac and Windows binary packages. But maybe not needed yet and can be changed later if needed.

Reply:
Yea it sound reasonable that any extensions we find in the package path should be loadable!